backend/src/route.rs

```diff
@@ -115,6 +158,10 @@ impl Router {
             }
         }
         pts.dedup();
-        Some(LineString::new(pts))
+        LineString::new(pts)
+    }
+
+    pub fn crosses_road(&self, road: RoadID) -> bool {
```

Comment:
To make sure I'm understanding our terminology: a RoadID corresponds to a segment of the road network which has no intersections in its interior. Is that right?

So there's no way to "cross" a road without traveling along it. 

In common language, traveling "through" an intersection would mean traveling from one RoadID and then switching to another RoadID at the point of the intersection (even though in common language we'd call it the same "road" all along).

The above is all totally fine - just check if I'm misunderstanding.

Reply:
All correct. It will get a bit more complicated for #30, because the transition from one Road to another Road within an Intersection will have to be modeled explicitly. (Called "turn" or "movement" in abstreet.) Any ideas for more clear terms, happy to use